### DISCUSSION TITLE
About prodigy plus optimizer

### DISCUSSION THREAD

**Discussion by NaotoEnjoyer** [67372a/LoRA_Easy_Training_Scripts#54] [General]
hey i wonder if theres some other prodigy plus optimizers? i heard about it but i see in your fork only the prodigy plus schedule free, is it meant to be only schedule free? or theres base prodigy plus too? i heard plus version is quite a bit better than base prodigy so i wanted to try it but found only this one which is kinda bad tbh since prodigy might rly use rex scheduler to avoid near end convergence prodigy fucks up a bit

**Comment by 67372a** [comment]
Hello @NaotoEnjoyer , ProdigyPlusScheduleFree is the only one, https://github.com/LoganBooker/prodigy-plus-schedule-free, if using schedulefree, it is advised to use the constant schedule, schedulefree can be disabled as per the repo documentation for the optimizer

**Reply by visenzero** [reply]
Hi there!

Do you maybe have any example settings of prodigy plus that you could share or some other ones that you found very good?
There are so many cool looking Optimizers that I don't know where to begin haha.

Sorry for the issue, I would just really appreciate it.

**Reply by 67372a** [reply]
Hey @visenzero , I don't really leverage prodigy plus myself, so can't as advise too much on it, I'd just start with its defaults and tune it. I'd consider EDM2, differential guidance, lora plus (assuming lycoris lora,locon,glora), and leaning towards logit normal timestep sampling vs uniform for SDXL.

I'll share a baseline config later today.

**Reply by visenzero** [reply]
Thank you this would help me soooo much, you are a great guy, thanks man!
I mostly train loras based on style, but anything would be amazing and helpful

**Reply by 67372a** [reply]
@visenzero 
[tezy8horni_ocgopt_glora_ortho_bs8_ga_2_v0_1.toml.txt](https://github.com/user-attachments/files/23918477/tezy8horni_ocgopt_glora_ortho_bs8_ga_2_v0_1.toml.txt)

Doesn't apply use_ram_ramtorch network due to some else reporting issues with training using it, but haven't yet looked into if it's a specific interaction with something else, as some others haven't had issues. Not to be considered a gold standard, but a starting point.
